rollup.config.ts

```diff
@@ -157,6 +163,7 @@ const buildIife = {
   },
   plugins: [
     replace({
+      preventAssignment: true,
```

Comment:
The next major version of the replace plugin will set this to `true` anyway. This is a recommended action from the plugin.